adhocteam/Head-Start-TTADP
# Fix region order and add non-grantees.

## Description of change
Follow-up PR to the TTA overview widget. Fixes region order in the region select dropdown. Adds non-grantees to the widget and combines duration into one item. Additionally, it includes the legacy reports in statistics and limits the reports to approved ones.


## How to test
- go to sandbox `https://tta-smarthub-sandbox.app.cloud.gov/activity-reports`
- assign yourself permissions to a lower number region (you might have to remove one first if having permissions to all)
- verify that the available regions in the region selectors are displayed in an ascending order
- add an activity report for a non-grantee
- verify that the widget was updated

This can also be tested locally.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-143


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
